latitude/index.html

```diff
@@ -0,0 +1,271 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Latitude</title>
+    <link
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.6.0/css/all.min.css"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    
+   
+      <main class="banner">
+        <navbar class="menu-bar">
+            <ul class="menu-items">
+              <li>
+                <a href="#"><i class="fa-solid fa-bars hamburger"></i></a>
+              </li>
+              <li class="item"><a href="#">Home</a></li>
+              <li class="item"><a href="#">Dropdown</a></li>
+              <li class="item"><a href="#">Left Sidebar</a></li>
+              <li class="item"><a href="#">Right Sidebar</a></li>
+              <li class="item"><a href="#">No Sidebar</a></li>
+              <li class="item"><a href="#">Elements</a></li>
+            </ul>
+          </navbar>
+          <article class="banner-content">
+        <header class="logo">
+          <span>Latitude</span>
+        </header>
+        <section class="banner-contentainer">
+          <h2>Magna phasellus consequat sed ipsum dolor</h2>
+          <p>
+            Feugiat cursus tempus adipiscing, phasellus sit amet lorem ipsum.
+          </p>
+        </article>
+    </section>
+      </main>
+      <article class="main-container">
+      <article class="poster-container">
+        <section class="poster">
+            <header class="poster-head">
+            <strong>Etiam sed feugiat</strong> <span class="poster-span">consequat amet lorem phasellus sed ultrices imperdiet dolore lacus
+                feugiat adipiscing magna consequat.</span>
+           
+            </header>
+          <hr />
+          <div class="poster-content">
+            <p>
+              Vestibulum viverra quam tortor ultricies convallis eros
+              condimentum. Vestibulum vulputate sagittis sem tincidunt
+              sustincidunt vestibulum congue lorem natoque penatibus et magnis
+              dis parturient.
+            </p>
+            <a href="#" class="btn">Continue Reading</a>
```

Comment:
check this button, for me it is like the following.
![image](https://github.com/user-attachments/assets/a2264ef0-7a31-477c-b36e-41acde9efa3f)